pom.xml

```diff
@@ -739,6 +741,64 @@
             </build>
         </profile>
         <profile>
+            <id>jbosseap-managed-7-1</id>
+            <properties>
+                <metamer.classifier>jee6</metamer.classifier>
+                <version.eap>property.version.eap.not.specified</version.eap>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-managed</artifactId>
+                    <version>2.0.0.Final</version>
+                    <scope>test</scope>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.wildfly.core</groupId>
+                    <artifactId>wildfly-controller-client</artifactId>
+                    <version>2.1.0.Final</version>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.wildfly.core</groupId>
+                    <artifactId>wildfly-cli</artifactId>
+                    <version>2.1.0.Final</version>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>org.jboss.logmanager</groupId>
+                            <artifactId>jboss-logmanager</artifactId>
+                        </exclusion>
+                    </exclusions>
+                </dependency>
+
+                <!-- probably a bug in wildfly-cli dependencies -->
+                <!-- not required if the dependency on wildfly-patching below is also added -->
+                <dependency>
+                    <groupId>org.jboss.logmanager</groupId>
+                    <artifactId>jboss-logmanager</artifactId>
+                    <version>2.0.3.Final</version>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemProperties>
+                                <arquillian.launch>jbosseap-managed-7-0</arquillian.launch>
+                                <!-- following is used in tests -->
+                                <JBOSS_HOME>${jbossEAP70Home}</JBOSS_HOME>
```

Comment:
I would either rename `jbossEAP70Home` to `jbossEAP7Home` and used it for both 7.0 and 7.1 or created a new property specific for EAP 7.1.